### PR TITLE
Weblate translations

### PR DESCRIPTION
**Pull request by renatolond**

### CODE REVIEW

File: config/locales/doorkeeper.pt.yml [hunk 1]
```diff
@@ -72,27 +73,30 @@ pt:
...
-        access_denied: The resource owner or authorization server denied the request.
-        credential_flow_not_configured: Resource Owner Password Credentials flow failed due to Doorkeeper.configure.resource_owner_from_credentials being unconfigured.
-        invalid_client: Client authentication failed due to unknown client, no client authentication included, or unsupported authentication method.
-        invalid_grant: The provided authorization grant is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.
-        invalid_redirect_uri: The redirect uri included is not valid.
-        invalid_request: The request is missing a required parameter, includes an unsupported parameter value, or is otherwise malformed.
-        invalid_resource_owner: The provided resource owner credentials are not valid, or resource owner cannot be found
-        invalid_scope: The requested scope is invalid, unknown, or malformed.
+        access_denied: O proprietário do recurso ou servidor de autorização negou o pedido.
+        credential_flow_not_configured: |-
```
Comment (ykzts) [inline config/locales/doorkeeper.pt.yml]: one line.

File: config/locales/doorkeeper.pt.yml [hunk 2]
```diff
@@ -72,27 +73,30 @@ pt:
...
+          As credenciais da palavra-passe do proprietário do recurso falhou devido a que
+          Doorkeeper.configure.resource_owner_from_credentials não foram configuradas.
```
Comment (ykzts) [inline config/locales/doorkeeper.pt.yml]: one line.

File: config/locales/simple_form.ar.yml [hunk 3]
```diff
@@ -2,57 +2,19 @@
...
-          one: <span class="name-counter">1</span> حرف متبقي
-        header: PNG, GIF or JPG. على الأكثر 2 ميغابيت . سوف يتم تصغيرها إلى 700x335px
-        locked: يتطلب منك الموافقة يدويا على كل طلب للإشتراك بحسابك و منشوراتك تعرض لمتابعيك فقط دون غيرهم
-        note: <span class="note-counter">%{count}</span> أحرف متبقية
-        setting_noindex: تمس ملفك العمومي الخاص بك وصفحات الحالة
-        setting_theme: تغير المظهر الذي يبدو عليه ماستدون عندما تقوم بتسجيل دخولك على أي جهاز.
-      imports:
-        data: ملف CSV تم تصديره من خادوم مثيل  آخر لماستدون
-      sessions:
-        otp: أدخل الرمز الثنائي من هاتفك أو استخدم أحد رموز الاسترداد.
```
Comment (ykzts) [inline config/locales/simple_form.ar.yml]: why deleting?

File: config/locales/simple_form.ar.yml [hunk 4]
```diff
@@ -2,57 +2,19 @@
...
-        favourite: إبعث بريداً إلكترونياً عندما يعجب احدهم بمنشورك
-        follow: إبعث بريداً إلكترونياً عندما يتبعك أحد
-        follow_request: إبعث بريداً إلكترونياً عند تلقيك طلب بالمتابعة
-        mention: إبعث بريداً إلكترونياً عندما يشار إليك
-        reblog: إبعث بريداً إلكترونياً عندما يُرقّي احدهم منشورك
-    'no': لا
-    required:
-      mark: "*"
-      text: مطلوب
-    'yes': نعم
```
Comment (ykzts) [inline config/locales/simple_form.ar.yml]: why deleting?

File: config/locales/simple_form.pt.yml [hunk 5]
```diff
@@ -2,49 +2,11 @@
...
-        digest: Enviar um email da actividade nesta instância
-        favourite: Enviar email quando alguém adiciona um post teu aos favoritos
-        follow: Enviar email quando alguém te segue
-        mention: Enviar email quando alguém te menciona
-        reblog: Enviar email quando alguém partilhar um post teu
-    'no': Não
-    required:
-      mark: "*"
-      text: necessário
-    'yes': Sim
```
Comment (ykzts) [inline config/locales/simple_form.pt.yml]: why deleting?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/locales/simple_form.ar.yml (hunk 4 above)
Comment (renatolond) [reply config/locales/simple_form.ar.yml]: Hm, usual weblate weirdness. I'll revert

**Review by ykzts — APPROVED** [review]

**Pull request merged**